jest.config.js

```diff
@@ -1,10 +1,10 @@
 module.exports = {
-    setupFiles: ['./testSetup.js'],
-    transform: { '^.+\\.js|jsx$': 'babel-jest' },
+    setupFiles: ['<rootDir>/src/tests/setup.js'],
+    transform: { '^.+\\.jsx?$': 'babel-jest' },
     collectCoverageFrom: ['src/**/*.{js,jsx}'],
     snapshotSerializers: ['enzyme-to-json/serializer'],
     moduleNameMapper: {
-        '\\.(jpg|jpeg|png|gif|svg)$': '<rootDir>/__mocks__/fileMock.js',
+        '\\.(jpg|jpeg|png|gif|svg)$': '<rootDir>/__mocks__/filemock.js',
```

Comment:
Why is the file changed to lowercase here?

Reply:
I will rectify that.